rts/Lua/LuaMathExtra.cpp

```diff
@@ -159,7 +159,8 @@ int LuaMathExtra::erf(lua_State* L) {
 }
 
 
-/*** Returns smoothstep(edge0, edge1, v).
+/*** Clamps and rescales v to a value between [0; 1] based on the edges and then applies the smoothstep function.
```

Comment:
You can add a one liner for the summary of what the method does above `@function`.

Below `@function` you can put the long description.

Reply:
e.g.

```
Clamps and rescales to a value between [0; 1]

@function bla

Clamps and rescales v to a value between [0; 1] based on the edges and then applies the smoothstep function.

For example math.smoothstep(10, 25, 15) is 0.259, because 15 is 0.333 of the way from 10 to 25, and smoothstep(0.333) is 0.259
````